### ISSUE TITLE
Specify the gremlin connection as environment variable

### ISSUE THREAD

**Issue by cscetbon** [prabushitha/gremlin-visualizer#12]
@prabushitha could you provide a way to specify those when starting the server and remove them from interface ? something like GREMLIN_SERVER and GREMLIN_PORT (defaulting to 8182).

**Comment by prabushitha** [comment]
The advantage of defining the port and server in the interface is that you can work with multiple graph databases at once. This was one of the use cases when designing this tool. Otherwise, you may have to spawn up several visualizers which makes it hard to switch between.

**Comment by cscetbon** [comment]
yeah I understand that, I'd like a way to do it in certain cases like mine.

**Comment by noliran** [comment]
@prabushitha perhaps we can have the default server be the value of the ENV variable? this way we can preserve the way of working with several servers + have the default one set right
